tests/handlers/test_typing.py

```diff
@@ -120,8 +120,6 @@ def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
 
         self.datastore = hs.get_datastores().main
 
-        self.datastore.get_destination_retry_timings = AsyncMock(return_value=None)
-
```

Comment:
Was this just unused?

Reply:
Yeah, it must have broken the test somehow but I've forgotten how